test/blackbox/common/BlackboxTestsDiscovery.cpp

```diff
@@ -1319,4 +1325,38 @@ TEST(Discovery, ServerClientEnvironmentSetUp)
 
     ASSERT_TRUE(load_environment_server_info(text, output));
     ASSERT_EQ(output, standard);
+
+    // 11. Check loading from environment file
+    std::string filename = "../unittest/utils/environment_test_file.json";
```

Comment:
Better for this test to have its own configuration file, so that both tests (this blackbox and the unit test) become independent and can evolve separately.

Reply:
Done in c3bea832946981b011fe8800677087471a68e1e1.

Good suggestion. As the environment file could be changed at runtime we would need to add a blackbox test to check this feature and could directly use this same file instead of modifying the one included in the unit test folder.